packages/dev/docs/pages/releases/2024-09-30.mdx

```diff
@@ -0,0 +1,99 @@
+{/* Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+
+description: Happy fall! 🍁 We’re excited to announce that our color picker components are now generally available! We've also added a pending state for buttons in React Aria Components, advanced TreeView to beta, and added new Disclosure and DisclosureGroup components in alpha for building accordions and collapsible panels.
+
+date: 2024-09-30
+---
+
+# September 30, 2024 Release
+Happy fall! 🍁 We’re excited to announce that our color picker components are now generally available! This includes [ColorPicker](https://react-spectrum.adobe.com/react-aria/ColorPicker.html), [ColorArea](https://react-spectrum.adobe.com/react-aria/ColorArea.html), [ColorField](https://react-spectrum.adobe.com/react-aria/ColorField.html), [ColorSlider](https://react-spectrum.adobe.com/react-aria/ColorSlider.html), [ColorSwatch](https://react-spectrum.adobe.com/react-aria/ColorSwatch.html), [ColorSwatchPicker](https://react-spectrum.adobe.com/react-aria/ColorSwatchPicker.html), and [ColorWheel](https://react-spectrum.adobe.com/react-aria/ColorWheel.html). These enable you to build fully customizable color pickers, including accessible color descriptions for screen reader support.
```

Comment:
```suggestion
Happy fall! 🍁 We’re excited to announce that our color picker components are now generally available! This includes [ColorPicker](https://react-spectrum.adobe.com/react-aria/ColorPicker.html), [ColorArea](https://react-spectrum.adobe.com/react-aria/ColorArea.html), [ColorField](https://react-spectrum.adobe.com/react-aria/ColorField.html), [ColorSlider](https://react-spectrum.adobe.com/react-aria/ColorSlider.html), [ColorSwatch](https://react-spectrum.adobe.com/react-aria/ColorSwatch.html), [ColorSwatchPicker](https://react-spectrum.adobe.com/react-aria/ColorSwatchPicker.html), and [ColorWheel](https://react-spectrum.adobe.com/react-aria/ColorWheel.html). These enable you to build fully customizable color pickers, and include accessible color descriptions for screen reader support.
```

Reply:
they don't allow you to include accessible descriptions, they do include them. 😄 maybe "which include" 🤷

Reply:
How about:

These components include accessible color descriptions for screen reader support and enable you to build fully customizable color pickers.